src/git/config.go

```diff
@@ -47,24 +48,24 @@ func Config() *Configuration {
 var currentDirConfig *Configuration
 
 // NewConfiguration provides a Configuration instance reflecting the configuration values in the given directory.
-func NewConfiguration(dir string) *Configuration {
+func NewConfiguration(shell command.Shell, dir string) *Configuration {
```

Comment:
Looks like the dir argument is no unused

Reply:
Good catch! I'm shocked that the compilers or linters didn't complain about it. Fixed in https://github.com/git-town/git-town/pull/1373